rpmio/argv.c

```diff
@@ -248,4 +248,29 @@ char *argvJoin(ARGV_const_t argv, const char *sep)
 
     return dest;
 }
-    
+
+ARGV_t argvFromVaList(const char *fmt, va_list ap)
+{
+    ARGV_t argv = argvNew();
```

Comment:
No need for argvNew(), argvAdd*() handles that as needed. Without this, an invalid fmt will return NULL which is what we probably want.

I think I'd also make this a private helper inside rpmscript.c, all of argv.* is considered deprecated going forward so maybe we shouldn't add more public APIs there.

Reply:
Oh, fair points, thanks. Will update the PR later.